orcid-web/src/main/resources/freemarker/manage.ftl

```diff
@@ -249,39 +243,6 @@
                         </div>
                     </td>
                 </tr>
-                <#if !profile.orcidInternal.preferences.notificationsEnabled>
```

Comment:
Hey @rcpeters not sure why you removed this? is this something we dont want anymore?